superset/connectors/sqla/models.py

```diff
@@ -1467,6 +1469,26 @@ class and any keys added via `ExtraCache`.
             extra_cache_keys += sqla_query.extra_cache_keys
         return extra_cache_keys
 
+    @event_logger.log_this
```

Comment:
can i add event log this way? want to monitor the duration for sanity check, in case it may affect explore view perf.

Reply:
What you care most is the actual check function, the part where we check version and skip the health check (happens for most page views) is known to be pretty trivial. So maybe use this code instead:

```python
with event_logger.log_context(action="datasource_health_check") as log:
    message = check(self)
    ...
```